docker-compose.yml

```diff
@@ -9,10 +9,17 @@ services:
     build: ./
     env_file:
       - .env
-    volumes:
-      - "./autogpt:/app"
-      - ".env:/app/.env"
+    # Only uncomment this if you really want to expose the host directory to container via bind mount (less safe)
+    # Note that any changes to files within the container need to be copied out via docker cp
+    # volumes:
+    #   - ".:/app"
     profiles: ["exclude-from-up"]
 
   redis:
-    image: "redis/redis-stack-server:latest"
+    # Use redis-stack instead of redis-stack-server as it also includes RedInsight which is useful for debugging (als see https://hub.docker.com/r/redis/redis-stack)
```

Comment:
This change makes sense

Reply:
With regard to the redis image change, what do you propose? Will you incorporate that in your own PR?

Reply:
no - keep it in here since you have done the work already. Just remove the other stuff (assuming you agree it's not needed) and then rename the PR